lib/server/routes/buckets.js

```diff
@@ -73,6 +76,13 @@ function BucketsRouter(options) {
     uploadsRepository,
     usersRepository
   );
+
+  this.bucketEntriesUsecase = new BucketEntriesUsecase(
```

Comment:
perhaps it's better to just have one usecase (BucketsUsecase) ?

Reply:
All the storage is 'bucket-centric', so I have avoided joining everything there due to the fact that anything could be placed there due to this fact, creating a very big use case. Gone for splitting use cases by model entities, trying to keep everything related to that entity as near and located as possible. Also, we can discuss the structure as this is just an attempt, while we try to stabilize bridge old codebase